### DISCUSSION TITLE
Integrating this script into an existing mod's runtime workflow

### DISCUSSION THREAD

**Discussion by Rithari** [CoolCat467/fix-lwjgl#27] [Q&A] [answered]
Hey there, 

First of all, thank you a lot for making this script! 

I've recently entered into a development team of an existing minecraft mod, running on Forge 1.16.5. Unfortunately, running my IntelliJ run script to start a minecraft instance, it errors out due to a difference in given and needed architecture of LWJGL. 

Now, I understand how your script works if it's placed as a wrapper script in a launcher, but, would you be able to help me integrate it into the workflow of an existing mod? I'm the only member that's on an ARM device (macos) so no one's had to deal with this before me. 

![CleanShot 2024-03-14 at 19 21 33@2x](https://github.com/CoolCat467/fix-lwjgl/assets/22421051/02f12c83-ddfc-449d-9849-a4e17226cffb)

If you're able to help, I'll be happy to provide the more necessary details in private.

Thank you!

**Comment by CoolCat467** [comment]
Sure, I think I might be able to help. Censor as needed, but would it be possible for your to post the full text of the second line (logging.console.level=debug ... etc) and your Environment variables: text boxes?
Not super familiar with whatever this application is to be honest, but what options are available when you press that + button in the top tabs area and what is in the Modify options drop-down?

**Reply by CoolCat467** [reply, marked as answer]
Fixing issue ended up to just use correct architecture of 1.8 JDK
